Translations/pl.po

```diff
@@ -401,14 +401,14 @@ msgstr ""
 
 #: Source/DiabloUI/hero/selhero.cpp:282 Source/DiabloUI/hero/selhero.cpp:285
 msgid "Enter Name"
-msgstr "Wpisz Imię"
+msgstr "Wpisz Nazwę"
```

Comment:
"Nazwa" seems to not be the best choice here, we're talking about _a name_ for a character, hence "imię" seems better as it relates to a character, to a person somewhat.

Reply:
We agree with you.